### PR TITLE
CHANGELOG:add wal write system call metrics observation

### PR DESCRIPTION
**Pull request by Akiqqqqqqq**
Please read https://github.com/etcd-io/etcd/blob/main/CONTRIBUTING.md#contribution-flow.

ref: 
- #17616
- #17615

### CODE REVIEW

**Review by fuweid — APPROVED** [review]
LGTM

File: CHANGELOG/CHANGELOG-3.5.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ Previous change logs can be found at [CHANGELOG-3.4](https://github.com/etcd-io/
 
 ### etcd server
 - Fix [LeaseTimeToLive returns error if leader changed](https://github.com/etcd-io/etcd/pull/17704).
+- Add [wal write system call metrics observation](https://github.com/etcd-io/etcd/pull/17616).
```
Comment (ahrtr) [inline CHANGELOG/CHANGELOG-3.5.md:11]:
```suggestion
- Add [metrics `etcd_disk_wal_write_duration_seconds`](https://github.com/etcd-io/etcd/pull/17616).
```

**Review by jmhbnz — APPROVED** [review]
LGTM - Thanks @Akiqqqqqqq

**Review by ahrtr — APPROVED** [review]

**Pull request merged**